src/sink/prom.rs

```diff
@@ -149,6 +149,8 @@ impl Sender {
                     label("container_id",    container.id.to_string());
                     label("container_name",  container.name.to_string());
                     label("container_image", container.image.to_string());
+                    label("namespace", container.namespace.to_string());
+                    label("pod", container.pod.to_string());
```

Comment:
I haven't given the Prometheus label schema a good thinking, but maybe these should be prefixed with `k8s_`?